app/src/main/java/com/example/criminalintent/CrimeFragment.kt

```diff
@@ -0,0 +1,79 @@
+package com.example.criminalintent
+
+import android.os.Bundle
+import android.support.v4.app.Fragment
```

Comment:
You can use `androidx.fragment.app.Fragment` here instead. I think this is deprecated, the support library fragments have been moved to the Jetpack versions(packages start with `androidx`)